common/religion/religions/titanic_group.txt

```diff
@@ -47,7 +47,7 @@
 
 	# Placeholder
 	traits = {
-		virtues = { generous compassionate calm }
+		virtues = { just diligent callous }
```

Comment:
callous can't be both a virtue and a sin
besides we've discussed why it shouldn't be a virtue in the first place

Reply:
Good spotting, I left that in the sins as well by accident.
We never agreed it shouldnt be though, I still think it fits much better than wrathful considering how the titanforged generally act

Reply:
Went with Virtues: ``just / diligent / stubborn``
vs Sins: ``arbitrary / lazy / fickle``